model/swagger.yml

```diff
@@ -1051,54 +545,120 @@ paths:
-        required: false
-        schema:
-          type: integer
-          default: 100
-      - name: fields
-        in: query
-        description: returned fields, syntax field0,field1 but does not impact vnd.nasa.pds.pds4+json and vnd.nasa.pds.pds4+xml output formats
-        required: false
-        schema:
-          type: array
-          items:
-            type: string
-      - name: sort
-        in: query
-        description: sort results, syntax asc(field0),desc(field1)
-        required: false
-        schema:
-          type: array
-          items:
-            type: string
-      - name: only-summary
-        in: query
-        description: only return the summary, useful to get the list of available properties
-        required: false
-        schema:
-          type: boolean
-          default: false
+      - $ref: "#/components/parameters/Fields"
+      - $ref: "#/components/parameters/Identified"
+      - $ref: "#/components/parameters/Limit"
+      - $ref: "#/components/parameters/Sort"
+      - $ref: "#/components/parameters/Start"
+      - $ref: "#/components/parameters/SummaryOnly"
 
 externalDocs:
   description: Find out more information on the PDS API Specification
   url: 'https://github.com/NASA-PDS/pds-api-core'
 components:
+  parameters:
+    Fields:
+      name: fields
+      in: query
+      description: |
+        syntax: fields=field1,field2,...
+
+        behavior: this parameter and the headder Accept: type determine what content is packaged for the result. While the types application/csv, application/kvp+json, and text/csv return only the fields requesteted, all of the other types have a minimal set of fields that must be returned. Duplicating a minimally required field in this parameter has not effect. The types vnd.nasa.pds.pds4+json and vnd.nasa.pds.pds4+xml have a complete set of fields that must be returned; meaning this parameter does not impact their content. When fields is not used, then the minimal set of fields, or all when minimal is an empty set, is returned.
+
+        notes: the blob fields are blocked unless specifically requrested and only for the */csv and application/kvp+csv types.
+      required: false
+      schema:
+        type: array
+        items:
+          type: string
+    Identified:
+      name: identifier
+      in: path
+      description: |
+        syntax: lidvid or lid
+
+        behavior (lid): returns one or more items whose lid matches this lid exactly. If the endpoint ends with the identifier or /latest then a signle result is returned and it is the highest version. If the endpoint ends with /all then all versions of the lid are returned.
+
+        behavior (lidvid): returns one and only one item whose lidvid matches this lidvid exactly.
+
+        note: the current lid/lidvid resolution will match all the lids that start with lid. In other words, it acts like a glob of foobar*. It behaves this way from first character to the last
```

Comment:
To simplify should we say the current lidvid resolution ?

What about the lid resolution, if we search for lid "xx:yy:zz" would we get lidvid "xx:yy:zzAA::v2" ?

Maybe to be more concrete for PDS users, we can replace `foobar*` by `lid::*` if that is what is actually done by the code ?

Reply:
Ot maybe we should remove that note. That is confusing me...

Reply:
No, we should fix the notes in another ticket on a another day. They are there more or less to remind us that things are not working perfectly and need attention. They are not meant to clarify bad behavior to the end user. However, until they are fixed the end user might as well know about them so they do not report it as an error again.